app/javascript/controllers/share_controller.js

```
@@ -0,0 +1,75 @@
+        if (navigator.share) {
+            try {
+                await navigator.share({ title, text, url });
+                this.flashToast();
+                return;
+            } catch (err) {
+                const name = err && err.name;
+                if (name === 'AbortError' || name === 'NotAllowedError' || name === 'InvalidStateError') {
+                    return;
+                }
+
+            }
```

ESLint

[error] Block must not be padded by blank lines.

(padded-blocks)